src/ProjectNeon/Assets/Scripts/KeysEnum.cs

```diff
@@ -0,0 +1,7 @@
+﻿public enum KeysEnum
+{
+    MusicVolume,
+    SoundVolume,
+    CurrentHero,
```

Comment:
We will never store game state in the PlayerPrefs. Only options. 

No need for `CurrentHero`.